.github/workflows/data-json-contract.yml

```diff
@@ -0,0 +1,26 @@
+name: data-json-contract
```

Comment:
I think we might add more checks in future beside data-json-contract. But we can rename it later if needed.

Reply:
fixed